xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -1473,5 +1477,56 @@ test_build:
         </listitem>
       </itemizedlist>
     </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.use_cases.config_url_and_placeholder_variables">
+      <title>Using a Custom Configuration File URL in Combination with Placeholder Variables</title>
+
+      <para>It may happen that you have multiple repositories following the same
+      set of steps, and you would rather have one copy of the configuration
```

Comment:
```suggestion
      set of workflow steps, and you would rather have one copy of the configuration
```

Reply:
I know we sometimes just talk about "steps", but I would mention "workflows" once just to be extra clear